edward/inferences.py

```diff
@@ -376,7 +368,7 @@ def __init__(self, model, data=Data(), params=None):
 
     def build_loss(self):
         x = self.data.sample(self.n_data)
-        z, _ = self.variational.sample()
+        z = self.variational.sample()
         self.loss = tf.squeeze(self.model.log_prob(x, z))
```

Comment:
okay thank you.

minor suggestion: rename `z` to something like `point_estimate_z`?